test_workflows.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# if GALAXY_URL and GALAXY_USER_KEY are not set, try to set them from local untracked file
+[ ! $GALAXY_URL ] || [ ! $GALAXY_USER_KEY ] && source .secret.env
+export GALAXY_URL GALAXY_USER_KEY
+
+virtualenv -p python3 .venv; . .venv/bin/activate
+pip install -r requirements.txt
+
+# initalise parsec with url and api key.  Remove any existing parsec configuration stored in the home directory
+rm -f ~/.parsec.yml ||:
```

Comment:
I don't think this is good, if anyone runs this locally it'll wipe out their configuration. On the server we set

```
PARSEC_GLOBAL_CONFIG_PATH
```

to a pre-built parsec yml file. So you can remove lines 3-5, 11-12 probably

Reply:
Ah, so usegalaxy.org.au has a prebuilt yaml file for this.  I was wondering how it worked.

Reply:
Too much magic, I know :(

Reply:
parsec is great.  I hadn't seen it until today.  I'll start using it instead of writing little python scripts for making bioblend calls :-)